### ISSUE TITLE
[1] error Command failed with exit code 3221226505.

### ISSUE THREAD

**Issue by sontek** [agracio/electron-edge-js#46]
Hey, I just installed `electron-edge-js` for the first time and put this at the top of my electron app to see if it works:

```const edge = require('electron-edge-js');

var helloWorld = edge.func(function () {/*
    async (input) => {
        return ".NET Welcomes " + input.ToString();
    }
*/});
```

now when I start electron it just says:

```[1] error Command failed with exit code 3221226505.```

I'm using :
Electron v5 and Windows 10

**Comment by sontek** [comment]
The code isn't necessary, just:

```
var edge = require('electron-edge-js');
```

will crash it with the same exit code.

**Comment by ReOrg360** [comment]
+1. I am also getting the same error with electron 5 and Windows 10. It works fine with electron 4 but error comes back as soon as I upgrade to electron v5. I am willing to dig into code and help to fix if I receive some pointers.

**Comment by ReOrg360** [comment]
Good news!

Using following line of code before require('electron-edge-js'); fixed the issue for me:

`
process.env.EDGE_USE_CORECLR = 1;
`

cc @sontek

**Comment by sontek** [comment]
Nice, I’ll try it in 30min and let you know

**Comment by agracio** [comment]
Thats why there is a link to https://github.com/agracio/electron-edge-js-quick-start in READMe ;)

**Comment by ReOrg360** [comment]
Sorry for highjacking the thread earlier. My previous comment only made the error message go away. But as I checked in detail, I realized I don't want to use CORECLR. So I have removed the line process.env.EDGE_USE_CORECLR = 1; and I am back to seeing the error message again. :(

Latest version of electron-edge-js still works fine if I am using electron 4, but the error is for electron 5.

Following require statement fails in ".\node_modules\electron-edge-js\lib\edge.js" file.

`edge = require(edgeNative);`


If I am using Electron 4, the value of edgeNative is (Works fine):

` .\node_modules\electron-edge-js\lib\native\win32\x64\10.11.0\edge_nativeclr`

If I am using Electron 5, the value of edgeNative is (Fails):

`.\node_modules\electron-edge-js\lib\native\win32\x64\12.0.0\edge_nativeclr`
